docs/pages/guides/config-plugins.mdx

```diff
@@ -589,11 +608,14 @@ If you aren't comfortable with setting up a monorepo, you can try manually runni
 
 ### Modifying the AndroidManifest.xml
 
-Packages should attempt to use the built-in **AndroidManifest.xml** [merging system](https://android-doc.github.io/tools/building/manifest-merge.html) before using a config plugin. This can be used for static, non-optional features like permissions. This will ensure features are merged during build-time and not prebuild-time, which minimizes the possibility of users forgetting to prebuild. The drawback is that users cannot use [introspection](#introspection) to preview the changes and debug any potential issues.
+Packages should attempt to use the built-in **AndroidManifest.xml** [merging system](https://android-doc.github.io/tools/building/manifest-merge.html)
```

Comment:
```suggestion
Packages should attempt to use the built-in **AndroidManifest.xml** [merging system](https://developer.android.com/studio/build/manage-manifests)
```

The link above doesn't work anymore. We should replace it with the official Android docs.

Reply:
Great catch, thank you! 🙏